aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -94,7 +103,7 @@ class QLinearPackWeightInt8 final : public c10::OperatorKernel {
 };
 
 static auto registry = c10::RegisterOperators().op(
-    "quantized::linear_prepack(Tensor W) -> Tensor W_prepack",
+    "quantized::linear_prepack(Tensor W, Tensor? B) -> Tensor W_prepack",
```

Comment:
can you specify default value? that way we don't need to pass it in python all the time as None